Robot Code/src/org/usfirst/frc/team3316/robot/humanIO/SDB.java

```diff
@@ -160,11 +163,38 @@ else if (type == Boolean.class)
 
 	private void initSDB()
 	{
-		SmartDashboard.putData(new UpdateVariablesInConfig()); // NEVER REMOVE
-																// THIS COMMAND
+		SmartDashboard.putData(new UpdateVariablesInConfig()); //NEVER REMOVE THIS COMMAND
+		SmartDashboard.putData(new TankDrive());
 
 		/*
 		 * Remove these after finishing testing on prototype
+		 */		
+		putConfigVariableInSDB("chassis_CrossDefense_Pid_Kp");
```

Comment:
When are we using a PID controller in crossing a defense? Delete these variables if they are not needed.